website/content/contributors/getting-started.md

```diff
@@ -0,0 +1,111 @@
+
+
+#### Issues
+Issues are used to track bugs, feature requests, and other tasks in a project. When you find a bug or want to propose a new feature, you can create an issue in the project's issue tracker. This allows you to collaborate with other contributors and track the progress of your task. Many open source projects have "Good first issue" or "Help wanted" labels attached to certain issues, which are suitable for new contributors or those looking to make their first contribution. 
+
+These issues are typically well-defined and relatively easy to implement, and can provide a great opportunity for contributors to get started with the project and learn the workflow. Look for these labels in the issue tracker and consider starting with them if you are new to the project or open source contribution in general.
+
+#### Pull Requests
+Pull requests are a way to propose changes to a project. When you submit a pull request, you're asking the project maintainers to review your code or document changes and consider merging it into the main codebase. To create a pull request, you'll need to fork the project, make your changes, and then submit the pull request through the project's website or Git hosting platform.
+
+Code reviews are an essential part of the software development process as they help catch potential issues before they become problems. It's important to remember that code reviews are not personal and should not be taken as criticism of the contributor. Instead, they provide an opportunity for contributors to learn from each other and improve their skills. Code reviews can be especially beneficial for beginner contributors as they can learn from more experienced contributors about potential issues with their code and how to improve it. Overall, code reviews are a valuable mechanism for learning and improving code quality. Code is about working together to produce the best code or documentation.
+
+It's important to keep in mind that things might take some time when it comes to code reviews. Asynchronous communication is the norm for code reviews, and it can take time for reviewers to provide feedback. Additionally, there may be factors outside of the review process that can impact response times, such as timezone differences, country-specific holidays, and team stress related to releases or deadlines. It's not uncommon for contributors to become frustrated when they are unaware of these factors and feel that they are not being taken seriously. However, it's essential to maintain open communication and trust between contributors and reviewers, as this can help alleviate any misunderstandings and ensure that the review process remains productive and efficient.
```

Comment:
```suggestion
It's important to remember that things might take some time when it comes to code reviews. Asynchronous communication is the norm for code reviews, and it can take time for reviewers to provide feedback. Additionally, factors outside the review process may impact response times, such as timezone differences, country-specific holidays, and team stress related to releases or deadlines. It's not uncommon for contributors to become frustrated when they are unaware of these factors and feel that they are not being taken seriously. However, it's essential to maintain open communication and trust between contributors and reviewers, as this can help alleviate any misunderstandings and ensure that the review process remains productive and efficient.
```